views/home/index.hbs

```diff
@@ -1,2 +1,4 @@
+<center>
```

Comment:
the <center> tag is not supported in HTML5 - centering should be managed by css properties instead?

Reply:
Can you help us implement that, thanks.